### PR TITLE
DEV-3883-subject for Exercise is same string

### PR DESCRIPTION
**Pull request by mohamedLazyBob**
resolves 01-edu/public#1157

### CODE REVIEW

File: subjects/issamestring/README.md [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+```go
+package main
+
+import (
+	"fmt"
+	"piscine"
+)
+
+func main() {
+  fmt.Println(piscine.IsSameString("hello world!", "HELLO WORLD!"))
```
Comment (Hamzaelkhatri) [inline subjects/issamestring/README.md:30]: add a tabulation here

**Review by Hamzaelkhatri — APPROVED** [review]
looks good to me 💯

**Review by zainabdnaya — COMMENTED** [review]
All Good

File: subjects/issamestring/README.md (hunk 1 above)
Comment (mohamedLazyBob) [reply subjects/issamestring/README.md]: Done

**Review by Hamzaelkhatri — APPROVED** [review]

**Review by zainabdnaya — APPROVED** [review]

**Review by zainabdnaya — APPROVED** [review]
All Good

File: subjects/issamestring/README.md [hunk 2]
```diff
@@ -0,0 +1,43 @@
+## issamestring
+
+### Instructions
+
+Write a function that returns `true` if the two `strings` passed as parameter are equal. Otherwise it returns `false`.
+
+The function will ignore the case of the characters.
```
Comment (tiagocollot) [inline subjects/issamestring/README.md:7]: add missing "-" before the sentence.

File: subjects/issamestring/README.md [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+
+```go
+func IsSameString(s1, s2 string) bool {
+
+}
+```
+
+### Usage
+
+Here is a possible program to test your function :
```
Comment (tiagocollot) [inline subjects/issamestring/README.md:19]: Remove space in between 'function' and ':'

File: subjects/issamestring/README.md [hunk 4]
```diff
@@ -0,0 +1,43 @@
...
+)
+
+func main() {
+	fmt.Println(piscine.IsSameString("hello world!", "HELLO WORLD!"))
+	fmt.Println(piscine.IsSameString("0101 is awesome", "0101 is AWESOME"))
+	fmt.Println(piscine.IsSameString("foo baz", "foo bar"))
+}
+```
+
+And its output :
```
Comment (tiagocollot) [inline subjects/issamestring/README.md:36]: Remove space between 'output' and ':'

**Review by tiagocollot — CHANGES_REQUESTED** [review]

File: subjects/issamestring/README.md (hunk 3 above)
Comment (jotapero) [reply subjects/issamestring/README.md:19]: fixed

File: subjects/issamestring/README.md (hunk 4 above)
Comment (jotapero) [reply subjects/issamestring/README.md]: fixed

File: subjects/issamestring/README.md (hunk 2 above)
Comment (tiagocollot) [reply subjects/issamestring/README.md:7]: This issue was not addressed.

**Pull request merged**